.github/boring-cyborg.yml

```diff
@@ -152,6 +152,18 @@ firstPRWelcomeComment: |
   If you have any issues or are unsure about any anything please check our
   Contribution Guide (https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst)
 
+  Here are some useful points:
+  - Check the quality of your code (flake8, pylint and type annotations). Our [pre-commits](
+  https://github.com/apache/airflow/blob/master/STATIC_CODE_CHECKS.rst#prerequisites-for-pre-commit-hooks)
+  will help you with that.
+  - In case of a new feature add useful documentation (in docstrings or in `docs/` directory).
+  Adding a new operator? Consider adding an example DAG that shows how users should use it.
+  - Consider using [Breeze environment](https://github.com/apache/airflow/blob/master/BREEZE.rst), it’s a
+  heavy docker but it ships with a working Airflow and a lot of integrations.
+  - Be patient and persistent.
```

Comment:
```suggestion
  - Be patient and persistent. It might take some time to get a review or get the final approval from Committers. 
```

Reply:
take some time -> take some time and a number of iterations